packages/components/src/icon-button/index.js

```diff
@@ -56,7 +56,7 @@ function IconButton( props, ref ) {
 			className={ classes }
 			ref={ ref }
 		>
-			{ isString( icon ) ? <Dashicon icon={ icon } ariaPressed={ ariaPressed } /> : icon }
+			<Icon icon={ icon } ariaPressed={ ariaPressed } />
```

Comment:
This line breaks the web because ariaPressed is not a valid react prop in the web. Dashicon addresses t his properly but not the Icon component.

Reply:
This PR removes this `ariaPressed` https://github.com/WordPress/gutenberg/pull/17356
It's practically ready to merge.

Reply:
Cool, thank you

Reply:
Thanks for the heads up! I'll let you know when it gets merged.